Sources/Widgets/Widgets3D/LineWidget/behavior.js

```diff
@@ -219,7 +329,7 @@ export default function widgetBehavior(publicAPI, model) {
   // --------------------------------------------------------------------------
 
   publicAPI.handleLeftButtonRelease = () => {
-    if (isDragging && model.pickable) {
+    if (model.widgetState.getIsDragging() && model.pickable) {
```

Comment:
isDragging does not belong to the widget state. It is a behavior property. It can go into model.isDragging